### DISCUSSION TITLE
Intellisense shows loading indefinitely

### DISCUSSION THREAD

**Discussion by waleedcj** [wevm/wagmi#1107] [Q&A] [answered]
**Hi I am coming across really wired bug when in typeScript, that when I define a contract hook from wagmi any except "useContractRead" it will bug and vs code Intellisense will not work on the whole directory, here are some screenshots** 

![intestile stops](https://user-images.githubusercontent.com/35780884/196416292-1b4a48c2-875d-4867-baf4-6e7aa721015e.PNG)

![bug](https://user-images.githubusercontent.com/35780884/196416295-11aead77-42b6-4a1a-b941-8af2b07770fe.PNG)

ethers and wagmi are up to-date 
![package](https://user-images.githubusercontent.com/35780884/196416558-9c44de5a-0d15-4483-a6ec-70d9f6e7a285.PNG)

using version 4.8.4 typeScript React

**Comment by tmm** [comment]
Can you put together a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) (e.g. does it seem to be some specific code causing this)?

**Reply by waleedcj** [reply]
Hi, I've complied a sandbox it seems to work fine here https://codesandbox.io/s/lucid-microservice-ht38o5?file=/src/App.tsx

However, upon creating a new react app using CRA typescript and only adding wagmi and ethers the issue is still there my package.JSON the error is still there
```JSON
{
  "name": "wagmii",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@testing-library/jest-dom": "^5.14.1",
    "@testing-library/react": "^13.0.0",
    "@testing-library/user-event": "^13.2.1",
    "@types/jest": "^27.0.1",
    "@types/node": "^16.7.13",
    "@types/react": "^18.0.0",
    "@types/react-dom": "^18.0.0",
    "ethers": "^5.7.1",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-scripts": "5.0.1",
    "typescript": "^4.4.2",
    "wagmi": "^0.7.5",
    "web-vitals": "^2.1.0"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}


```
**as seen in the snippet IntelliSense status is still pending in the bottom bar**
![bugs](https://user-images.githubusercontent.com/35780884/196466656-bd59c885-2e4c-4acf-8c82-9c5fbefbb5a6.PNG)





**Also, I always switch between the versions when this happens**

![version](https://user-images.githubusercontent.com/35780884/196465695-1dd9f6ee-b553-4987-a4dd-8e8f9c9729de.PNG)



**I've also pushed this repo to github right here** https://github.com/waleedcj/wagmi 
**My coworker could also reproduce this issue on her vscode**

Tried to update @types/react, restart the typescript server does not fix, **can the devs do something?** ;)

**Reply by tmm** [reply]
Thanks looking into it!

**Comment by zouhangwithsweet** [comment]
same issue with me

**Reply by zouhangwithsweet** [reply]
`v0.6.6` is ok, up to `v0.7.5` is not woring.
https://stackblitz.com/edit/github-sedgph

**Reply by zouhangwithsweet** [reply]
`v0.6.8` is ok too. `> 0.7.x`is not woring.

**Reply by waleedcj** [reply]
did not try, but will do it, does it make a problem with the latest versions of @types/react?

**Reply by zouhangwithsweet** [reply]
Before import `useContractReads`, every thing is ok
<img width="779" alt="image" src="https://user-images.githubusercontent.com/21095710/196916132-fb415be8-c92f-46ae-9ad2-3134fa1cabee.png">

After imported, :) Typescript service has no responding. Funny issue
<img width="613" alt="image" src="https://user-images.githubusercontent.com/21095710/196916576-08a32e7b-6e61-449e-887d-2b9cba7cbd16.png">

**Reply by zouhangwithsweet** [reply]
Just use the [`official vite react/typescript template`](https://github.com/vitejs/vite/blob/main/packages/create-vite/template-react-ts/package.json)

**Comment by tmm** [comment, marked as answer]
Fixed in [`wagmi@0.7.6`](https://github.com/wagmi-dev/wagmi/releases/tag/wagmi%400.7.6)

**Reply by codescrafter** [reply]
still happening in @1.4.7

**Reply by tmm** [reply]
Can you create an issue with a minimal reproduction if it's still happening?
